apps/DisplayCluster/MasterWindow.cpp

```diff
@@ -420,6 +428,7 @@ void MasterWindow::loadState( const QString& filename )
         QMessageBox::warning( this, "Error", "Could not load state file.",
                               QMessageBox::Ok, QMessageBox::Ok );
     }
+    showWindowTitlesAction_->setChecked( displayGroup_->getShowWindowTitles() );
```

Comment:
and with the connection, this line can be removed and you can also remove showWindowTitlesAction_ from the class variables.